diff --git a/src/RuntimeOptions.h b/src/RuntimeOptions.h
index d281ef39..358a2efa 100644
--- a/src/RuntimeOptions.h
+++ b/src/RuntimeOptions.h
@@ -46,7 +46,7 @@ class RuntimeOptions {
         uint32_t popFailRecoveryMasters();
         void checkAndCrashCoordinator(const char *crashPoint);
 
-    PRIVATE:
+    PUBLIC:
         /**
          * Interface for all configuration option parsers. Generally
          * parsers should subclass this and add a constructor which
@@ -60,6 +60,7 @@ class RuntimeOptions {
             virtual ~Parseable() {}
         };
 
+    PRIVATE:
         void registerOption(const char* option, Parseable* parser);
 
         /**
